src/frontend/ProductFilter/ProductFilter.jsx

```diff
@@ -0,0 +1,71 @@
+import React from "react";
+import "./ProductFilter.css";
+
+const ProductFilter = () => {
+  return (
+    <div>
+      <div className="filter-container">
+        <div className="filter-content-container">
+          <filter-section className="flex-row flex-space-btw">
+            <h2>Filters</h2>
+            <button>clear</button>
+          </filter-section>
+
+          <filter-section>
+            <h2>Price</h2>
+            <div className="price-range flex-row">
+              <span>50</span>
+              <span>150</span>
+              <span>200</span>
+            </div>
```

Comment:
suggestion: you can add some gap between the elements.

![image](https://user-images.githubusercontent.com/57091383/162577119-a55c2e9e-5315-4915-9bf4-636617c2295c.png)